helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -61,8 +66,24 @@ internal class SplitCommand : CliktCommand(
             getSplitCurationFile(outputCurationsDir, it.id, inputCurationsFile.extension)
         }
 
+        val mapper = createBlockYamlMapper()
         groupedCurations.forEach { (outputFile, curations) ->
-            outputFile.writeValue(curations)
+            curations.forEach { originalCuration ->
+                val comment = originalCuration.data.comment?.wrapAt(COMMENT_WRAP_COLUMN)
```

Comment:
Suggestion: It may be slightly more concise to move the conversion of the comment to an extension function on the curation.